### PR TITLE
Add qt5 support fix

### PR DESCRIPTION
**Pull request by jonaslindemann**
Found some issues with the scrollwheel api.

### CODE REVIEW

File: backends/qtcommon.py [hunk 1]
```diff
@@ -142,12 +144,22 @@ def mouseMoveEvent(self, event):
...
+            numDegrees = event.angleDelta() / 8.0            
+        else:
+            numDegrees = event.delta() / 8.0
+                                    
         numSteps = numDegrees / 15.0
-        if event.orientation() == QtCore.Qt.Vertical:
+        
+        if qtlib == "pyqt5":
             horizontal, vertical = 0, numSteps
         else:
```
Comment (almarklein) [inline backends/qtcommon.py]: ? No horizontal scrolling in qt5?

Comment (almarklein) [inline backends/qtcommon.py]: From the code below (`event.verticalSteps.y()`) it looks like numSteps is a special value for both x and y. Maybe the code here should be `horizontal, vertical = numSteps.x(), numSteps.y()`. Then we can leave `axes.py` alone,

Comment (jonaslindemann) [inline backends/qtcommon.py]: I couldn't find the same concept in PyQt5, but I will check. I will also go a little deeper in the scrollwheel handling.

Comment (jonaslindemann) [inline backends/qtcommon.py]: PySide2 should also be possible. Mostly I did this to be able to use it in WinPython with Qt5 support. I will check.

**Comment by jonaslindemann** [comment]
Fixed the wheel issues in axes.py and qtcommon.py.

**Pull request merged**